packages/suite/src/hooks/wallet/useRbfForm.ts

```diff
@@ -38,24 +38,24 @@ const getEthereumFeeInfo = (info: FeeInfo, gasPrice: string) => {
         fromWei(info.levels[0].feePerUnit, 'gwei'),
     ).integerValue(BigNumber.ROUND_FLOOR);
 
-    const getMinFee = () => {
+    const getFee = () => {
         if (minFeeFromNetwork.lte(current)) {
             return current.plus(1);
         }
         return minFeeFromNetwork;
     };
 
-    const minFee = getMinFee();
+    const fee = getFee();
 
     // increase FeeLevel only if it's lower than predefined
     const levels = getFeeLevels('ethereum', info).map(l => ({
         ...l,
-        feePerUnit: minFee.toString(),
+        feePerUnit: fee.toString(),
     }));
     return {
         ...info,
         levels,
-        minFee: minFee.toNumber(), // increase required minFee rate
+        minFee: current.plus(1).toNumber(), // increase required minFee rate
```

Comment:
isn't it what `getMinFee` should do in previous code?

if minimum fee from the network (from backend) is lower than `current` then use current + 1 otherwise use minimum which is greater than current?

Reply:
It should prefill the input with the correct amount (that was fixed) but it was also passed to the form error handler.

We want to always set the error handler to `current + 1` but the prefill with the network fee or `current + 1`.

Because if user used gas price `2` and wants to replace the tx and the network fee is let's say `15` than he can't try just `10` but amount more than `15`.

Reply:
ah, i get it now. i see that before t[his change](https://github.com/trezor/trezor-suite/pull/8787/files) minFee was indeed set current + 1